platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -452,6 +452,9 @@ public void postInit(final VirtualStateAccessor state) {
             // it is necessary to use the statistics object from the previous instance of the state.
             statistics = new VirtualMapStatistics(state.getLabel());
         }
+        // VM size metric value is updated in add() and remove(). However, if no elements are added or
+        // removed, the metric may have a stale value for a long time. Update it explicitly here
+        statistics.setSize(size());
```

Comment:
Is it the only metric that needs to be set after reconnect?

Reply:
I couldn't find any other metric to be updated